src/window/keyboard/settings.rs

```diff
@@ -1,8 +1,9 @@
 use super::KeyboardLayout;
 use crate::settings::FromValue;
 
+#[derive(SettingGroup)]
```

Comment:
Interesting to split these up. Was that a clippy suggestion?

Reply:
It was more about the order of the derivation. It wanted SettingsGroup to be resolved before the prefix was used.